index.js

```diff
@@ -1,14 +1,96 @@
 const express = require('express');
 const helmet = require('helmet');
-
+const knex = require('knex');
+const dbConfig = require('./knexfile');
+const { zooId404 } = require('./helpers/error');
 const server = express();
 
 server.use(express.json());
 server.use(helmet());
 
+const db = knex({ ...dbConfig.development });
```

Comment:
Why use spread here? I haven't seen this before so I don't know if there's a reason to get a clean copy of the development object. This is how I've seen it in the past, though:
```javascript
const db = knex(dbConfig.development);

```

Reply:
Hrm yeah probably unnecessary. Not sure what I was thinking.